Data/3_0/Skills/act_int.lua

```diff
@@ -6739,14 +6821,9 @@ skills["Stormbind"] = {
 	skillTypes = { [SkillType.Spell] = true, [SkillType.Area] = true, [SkillType.Hit] = true, [SkillType.AreaSpell] = true, [SkillType.Channelled] = true, [SkillType.LightningSkill] = true, [SkillType.SkillCanTotem] = true, [SkillType.Duration] = true, },
 	statDescriptionScope = "skill_stat_descriptions",
 	castTime = 0.12,
-	baseFlags = {
```

Comment:
This seems like a bad merge, with Spirit Offering and Stormbind mixing with each other.

Reply:
Looks like that's caused by a visual bug in GitHub, the actual changeset is correct.